SU-SWS/stanford_bean_types
# 7.x 2.x soe 3443 testi image

# NOT READY FOR REVIEW

# Summary
_See ticket:_ https://stanfordits.atlassian.net/browse/SOE-3443

# Needed By (Date)
- Sprint end
_For version release: https://stanfordits.atlassian.net/projects/SOE/versions_

# Criticality
- Client is eagerly awaiting this.
- Fixes broken stuff


# Steps to Test
- Checkout this branch: `git checkout 7.x-2.x-SOE-3443-testi-image`
- Rebuild registry and clear cache: `drush rr ; drush cc all`
- Revert feature: ` drush fr stanford_bean_types`
- Navigate to `catalog-patterns/blocks-and-beans/testimonial-blocks`
- Verify content appears as expected

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3443

## Related PRs

## More Information

## Folks to notify
`@minorwm `
`@annawatt `
`@kerri-augenstein `
`@boznik `

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
